self-call.p4(17): [--Werror=not-found] error: c: declaration not found
    c() c_inst;
    ^
